### ISSUE TITLE
Amazon MQ throughput-optimized message brokers

### ISSUE THREAD

**Issue by ewbankkit** [hashicorp/terraform-provider-aws#11325]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Amazon MQ supports throughput-optimized message brokers, backed by Amazon Elastic Block Store.

### New or Affected Resource(s)

* [`aws_mq_broker`](https://www.terraform.io/docs/providers/aws/r/mq_broker.html)

### Potential Terraform Configuration

```hcl
resource "aws_mq_broker" "example" {
  storage_type = "EBS"
}
```

### References

[Announcement](https://aws.amazon.com/about-aws/whats-new/2019/12/amazon-mq-introduces-throughput-optimized-message-brokers/).
[Developer guide](https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/broker-storage.html).

Requires [AWS SDK v1.26.3](https://github.com/aws/aws-sdk-go/releases/tag/v1.26.3):
* https://github.com/terraform-providers/terraform-provider-aws/pull/11314

**Comment by dds** [comment]
@ewbankkit please see @gdepuille's PR. It would be great if we could please get a status update on this issue.

**Comment by YakDriver** [comment]
Closed by #16108! 🎉
